vocabularies/UI.xml

```diff
@@ -1401,6 +1401,37 @@ The trend is
         <Annotation Term="Core.IsLanguageDependent" />
       </Term>
 
+      <Term Name="InputMask" Type="UI.InputMaskType" Nullable="false" AppliesTo="Property Parameter">
+        <Annotation Term="Core.Description" String="Properties or parameters annotated with this term will get a mask in edit mode." />
+        <Annotation Term="Core.LongDescription">
+          <String>Input masks not only improve readability, but also help to enter data correctly. 
+So, masks can be especially useful for input fields that have a fixed pattern, e.g. international bank account numbers or social security numbers. 
+[Here](../examples/UI.InputMask-sample.xml) you can find an example for this annotation</String>
+        </Annotation>
+      </Term>
+      <ComplexType Name="InputMaskType">
+        <Property Name="Mask" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="The mask to be applied to the property" />
+        </Property>
+        <Property Name="Placeholder" Type="Edm.String" DefaultValue="_">
+          <Annotation Term="Core.Description" String="A single character symbol that is used as a substitute." />
```

Comment:
A substitute for what? Can you give an example?

Reply:
This is the symbol that's shown where there is no user input, eg. DE__ ____ ____ ____ ____ __ for a german IBAN. Maybe substitute is not really the right choice for the explanation - do you have a better one?
Thanks!

Reply:
But the UI mask for IBAN would be `DE99 9999 9999 9999 9999 99` since the variable characters are digits.

For a Swiss IBAN, `CH99 9999 9999 9999 9999 99` would not work, because the leading `C` can mean _any_ letter, right?

The example is still unclear to me.

Reply:
I wrote something wrt the placeholder that's shown e.g. in a UI control, and you are writing about (the definition of) a mask. 
An IBAN in general is not well suited for a mask as e.g. the length differs based on country. If you want to prepare your UI for a swiss IBAN, you can define another mask and different rules.